packages/static-build/test/fixtures/10-gatsby-without-build-script/package.json

```diff
@@ -1,45 +0,0 @@
-{
-  "name": "10-gatsby-without-build-script",
-  "private": true,
-  "description": "A simple starter to get up and developing quickly with Gatsby",
-  "version": "0.1.0",
-  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
-  "engines": {
-    "node": "16.x"
-  },
-  "dependencies": {
-    "gatsby": "^2.32.3",
-    "gatsby-image": "^2.11.0",
-    "gatsby-plugin-manifest": "^2.12.0",
-    "gatsby-plugin-offline": "^3.10.0",
-    "gatsby-plugin-react-helmet": "^3.10.0",
-    "gatsby-plugin-sharp": "^2.14.1",
-    "gatsby-source-filesystem": "^2.11.0",
-    "gatsby-transformer-sharp": "^2.12.0",
-    "prop-types": "^15.7.2",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-helmet": "^6.1.0"
-  },
-  "devDependencies": {
-    "prettier": "^2.2.1"
-  },
-  "keywords": [
-    "gatsby"
-  ],
-  "license": "MIT",
-  "scripts": {
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Is it OK that we're no longer asserting this framework works without a "build" script?

I think so because the default build command is the same as what we test anyway. I just wanted to make sure this was done intentionally.

Reply:
Yep exactly